gnucash/gnucash-cli.cpp

```diff
@@ -100,7 +100,9 @@ Gnucash::GnucashCli::configure_program_options (void)
     bpo::options_description report_options(_("Report Generation Options"));
     report_options.add_options()
     ("report,R", bpo::value (&m_report_cmd),
-     _("Execute report related commands. Currently only one command is supported.\n\n"
+     _("Execute report related commands. Currently three commands are supported.\n\n"
```

Comment:
No need to count the commands if you have more than one. I added this because it felt a bit odd to make a list of only one command without hinting more would follow in the future. Now you have three, this can simply become
`The following commands are supported.`
And we can add as many as we like in the future without having to change this line ever again.